.gitignore

```diff
@@ -20,3 +20,6 @@ icechunk-python/uv.lock
 # pixi environments
 .pixi
 *.egg-info
+
+# Jetbrains IDE
+.idea/
```

Comment:
@ebarylko could you add an empty line at the end of this file? The linter doesn't like it otherwise. 

By the way, you can use `just lint` if you want to check these things locally.